src/MSAL.PCL/Internal/Requests/BaseRequest.cs

```diff
@@ -41,13 +42,17 @@ internal abstract class BaseRequest
         internal readonly AuthenticationRequestParameters AuthenticationRequestParameters;
         internal readonly Authority Authority;
         internal readonly TokenCache TokenCache;
+        protected Exception Exception;
```

Comment:
>protected Exception Exception; [](start = 8, length = 30)

can't we throw in the case of error instead of keeping the exception object around?